lld/COFF/InputFiles.h

```diff
@@ -132,16 +139,35 @@ class ArchiveFile : public InputFile {
   llvm::DenseSet<uint64_t> seen;
 };
 
+// A synthetic -start-lib/-end-lib archive.
+class CmdLineArchive : public InputFile {
+public:
+  explicit CmdLineArchive(SymbolTable &s, MemoryBufferRef m, StringRef startLib,
+                          StringRef endLib)
```

Comment:
Since the caller always passes an empty memory buffer, we might as well drop the `m` argument and hardcode it here.